app/assets/javascripts/preferences/panes/two-factor-auth/ScanQRCode.tsx

```diff
@@ -35,7 +38,7 @@ export const ScanQRCode: FunctionComponent<{
       <TwoFactorDialogDescription>
         <div className="flex flex-row gap-3 items-center">
```

Comment:
Have you tested the layout on the desktop application? I have recently registered a styling bug due to the `gap-x` classes, seems they're not working properly on Electron

Reply:
For some reason the build on desktop is failing. The errors I'm seeing are webpack css failures connected with reach components. I think that needs to be resolved as part of another task.